src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -17,7 +17,6 @@ import scala.concurrent.duration._
 import scala.util.control.NonFatal
 
 @IntegrationTest
-@UnstableTest
```

Comment:
I told you https://github.com/mesosphere/marathon/pull/5183#pullrequestreview-22304211